src/pages/index.js

```diff
@@ -51,17 +58,120 @@ const getHomepageConferences = (events = []) =>
       date: getFeaturedEventDate(n)
     }))
 
-const IndexPage = ({
-  data: { contentfulHomepage: content, allContentfulMeetupEvent: events },
-  location
-}) => {
+const StyledBlueBackground = styled(BlueBackground)`
+  margin-top: -${remcalc(36)};
+`
+
+const IntroRow = styled(Row)`
+  padding-top: ${({ theme }) => theme.space[4]};
+  padding-bottom: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[5]};
+    padding-bottom: ${({ theme }) => theme.space[6]};
+  `}
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+    padding-bottom: ${({ theme }) => theme.space[7]};
+  `}
+`
+
+const StyledCardTitle = styled(CardTitle)`
+  display: inline-block;
+  text-decoration: underline;
+`
+
+const IntroLinkWrapper = styled.div`
+  ${breakpoint('tablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+`
+
+const IntroSectionTitleWrapper = styled.div`
+  ${breakpoint('tablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+`
+
+const IntroImageWrapper = styled.div`
+  position: absolute;
+  max-width: 1500px;
+  bottom: 0;
+  left: 50%;
+  right: 0;
+  width: 100%;
+`
+
+const IntroImage = styled(Image)`
+  left: -50%;
+`
+
+const IntroSection = ({ illustration }) => {
+  const [first, toggle] = useState(true)
+
+  const introCopy = first
+    ? "We're a technology company that builds great technology  companies"
+    : 'Creating technology capabilities for you, that lasts beyond us.'
+
+  return (
+    <StyledBlueBackground style={{ position: 'relative' }}>
+      <IntroImageWrapper>
+        <IntroImage image={illustration.childImageSharp} />
+      </IntroImageWrapper>
+      <Grid>
+        <IntroRow style={{ position: 'relative' }}>
+          <Col width={[1, 1, 1, 1, 7 / 12]} style={{ position: 'relative' }}>
+            <IntroSectionTitleWrapper>
+              <SectionTitle reverse as="h1">
+                {introCopy}
+              </SectionTitle>
+            </IntroSectionTitleWrapper>
+            <Subtitle reverse muted>
+              What we offer
+            </Subtitle>
+            <IntroLinkWrapper>
```

Comment:
an improvement that I think would be nice is to iterate from an array of object
```
const services = [{
  title: 'Software engineering consultancy',
  to: '/engineering'
}]
```
and
```
{services.map(service =>
  <StyledCardTitle noPaddingTop as={Link} reverse to={service.to}>
    {service.name}
  </StyledCardTitle>
```
it will make it easily extendable when we have new services coming.

Reply:
This is now in contentful so am doing that!